src/encoded/types/antibody_lot.py

```diff
@@ -158,151 +162,166 @@ def lot_reviews(characterizations, targets, request):
+        'targets': sorted(targets),
+        'status': 'characterized to standards with exemption'
+        if is_control else ab_states[(None, None)],
+        'detail': 'IgG does not require further characterization.'
+        if is_control else ab_state_details[(None, None)]
+    }
+
     if not characterizations:
         # If there are no characterizations, then default to awaiting characterization.
-        return [{
-            'biosample_term_name': 'any cell type or tissues',
-            'biosample_term_id': 'NTR:99999999',
-            'organisms': sorted(target_organisms['all']),
-            'targets': sorted(targets),  # Copy to prevent modification of original data
-            'status': 'characterized to standards with exemption' if is_control else 'awaiting characterization',
-            'detail': 'IgG does not require further characterization.' if is_control else 'No characterizations submitted for this antibody lot yet.'
-        }]
+        return [base_review]
 
     review_targets = set()
-    primary_chars = []
-    secondary_chars = []
+    char_organisms = dict()
+    primary_chars = list()
+    secondary_chars = list()
     secondary_status = None
 
+    status_ranking = {
+        'characterized to standards': 15,
+        'characterized to standards with exemption': 14,
+        'partially characterized': 13,
+        'awaiting characterization': 12,
+        'not characterized to standards': 11,
+        'not pursued': 10,
+        'compliant': 9,
+        'exempt from standards': 8,
+        'not compliant': 7,
+        'pending dcc review': 6,
+        'in progress': 5,
+        'not reviewed': 4,
+        'not submitted for review by lab': 3,
+        'deleted': 2
+    }
+
     # Since characterizations can only take one target (not an array) and primary characterizations
     # for histone modifications may be done in multiple species, we really need to check lane.organism
     # against the antibody.targets.organism list to determine eligibility of use in that organism.
 
     for characterization_path in characterizations:
         characterization = request.embed(characterization_path, '@@object')
         target = request.embed(characterization['target'], '@@object')
-        organism = target['organism']
 
         # instead of adding to the target_organism list with whatever they put in the
         # characterization we need to instead compare the lane organism to see if it's in the
         # target_organism list. If not, we'll need to indicate that they characterized an
         # organism not in the antibody_lot.targets list so it'll have to be reviewed and
         # added if legitimate.
         review_targets.add(target['@id'])
-
+        char_organisms[characterization['@id']] = target['organism']
         # Split into primary and secondary to treat separately
         if 'primary_characterization_method' in characterization:
             primary_chars.append(characterization)
+
```

Comment:
Should maybe antibody_lot have 2 arrays (calculated props?), one for primary and one for secondaries?
I guess UI treats them similarly.